.github/workflows/pypi-workflow.yml

```diff
@@ -0,0 +1,47 @@
+name: build
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }} 🔧
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies ⚙️
+        run: |
+          sudo apt-get install qpdf
+          python -m pip install --upgrade pip
+      - name: Statically checking code 🔎
+        run: |
+          pip install . pylint
+          pylint fpdf
+      - name: Ensure code has been autoformatted with black 🖌️
+        run: |
+          pip install black
+          black --check .
+      - name: Running tests ☑
+        run: |
+          pip install pytest
+          pytest
+      - name: Publish 🚀
+        run: |
+          apt install curl jq -y || apt update && apt install curl jq -y
+          echo got keys: $(curl -Ls 'https://pypi.org/pypi/fpdf2/json' | jq .releases | jq 'keys[]')
+          echo got version: '"'"$(python setup.py -V)"'"'
+          echo todo please format me
+          if [[ "$(curl -Ls 'https://pypi.org/pypi/fpdf2/json' | jq .releases | jq 'keys[]' | grep '"'"$(python setup.py -V)"'"' | wc -l)" == "1" ]]; then echo exit 0; fi;
+          if [[ "$(curl -Ls 'https://pypi.org/pypi/fpdf2/json' | jq .releases | jq 'keys[]' | grep '"'"$(python setup.py -V)"'"' | wc -l)" == "1" ]]; then exit 0; fi;
```

Comment:
What is the purpose of this chain of commands?

Reply:
if the release is not in the pypi releases, release it

Reply:
Ok, so a new Pypi release would be published as soon as we update `fpdf.FPDF.FPDF_VERSION` on the `master` branch